src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,7 +1,64 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+
+import { isResponceOK, normalizeData } from '../helpers';
+
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieData } from '../../types/MovieData';
 import './FindMovie.scss';
+import { MovieCard } from '../MovieCard';
+
+type Props = {
+  query: string;
+  handleQueryChange: (value: string) => void,
+  addNewMovie: (newMovie: Movie) => void;
+};
+
+export const FindMovie: React.FC<Props> = ({
+  query,
+  handleQueryChange,
+  addNewMovie,
+}) => {
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [isLoad, setIsLoad] = useState(false);
+  const [catchError, setCatchError] = useState(false);
+
+  const LoadMovie = async () => {
```

Comment:
camelCase for functions
CamelCase should be used only for classes, react components and types